lib/cypress/patient_zipper.rb

```diff
@@ -7,24 +7,25 @@
 
 module Cypress
   class HTMLExporter
-    EXPORTER = HealthDataStandards::Export::HTML.new
+    # TODO: add HTML Export in CQM-Parsers
+    # EXPORTER = HealthDataStandards::Export::HTML.new
     attr_accessor :measures
 
     def initialize(measures, start_time, end_time)
-      @qdm_patient_converter = CQM::Converter::QDMPatient.new
+      # TODO: conversion will not be needed, QDM -> HTML
+      # @qdm_patient_converter = CQM::Converter::QDMPatient.new
       @measures = measures.to_a
       @start_time = start_time
       @end_time = end_time
     end
 
     def export(patient)
+      # TODO: add HTML Export in CQM-Parsers
       # TODO: R2P: make sure patient export works with HDS HTML exporter
-      hdsrecord = @qdm_patient_converter.to_hds(patient)
-      hdsrecord.bundle_id = patient.bundleId
-      # replace the end_time with nil if the end_time is > 1893474001, which is 1/1/2030.
-      # This will get the HTML exporter to export the end time as present.
-      hdsrecord.entries.each { |de| de.end_time = nil if de.end_time && de.end_time > 1_893_474_001 }
-      EXPORTER.export(hdsrecord, measures)
+      # hdsrecord = @qdm_patient_converter.to_hds(patient)
+      # hdsrecord.bundle_id = patient.bundleId
+      # EXPORTER.export(hdsrecord, measures)
```

Comment:
Commented out code (and TODOs) can be removed.

Reply:
Fixed